sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/LightwalletdServer.kt

```diff
@@ -0,0 +1,5 @@
+package cash.z.ecc.android.sdk.model
+
+data class LightwalletdServer(val host: String, val port: Int, val isSecure: Boolean) {
```

Comment:
I call this LightWalletEndPoint instead because we don't know if this is actually a server or many behind a load balancer.

Reply:
I renamed it.